presto-main/src/test/java/io/prestosql/execution/TestSqlTaskExecution.java

```diff
@@ -177,7 +178,7 @@ public void testSimple(PipelineExecutionStrategy executionStrategy)
 
             //
             // test body
-            assertEquals(taskStateMachine.getState(), TaskState.RUNNING);
+            assertTrue(taskStateMachine.getState().isRunning());
```

Comment:
please test actual states and don't use `isRunning()`. We need to have test where we validate `FLUSHING` state